lib/rester/client/adapters/local_adapter.rb

```diff
@@ -37,11 +37,13 @@ def _request(verb, path, opts={})
 
         response = Timeout::timeout(timeout) do
           service.call(
-            'REQUEST_METHOD' => verb.to_s.upcase,
-            'PATH_INFO'      => path,
-            'CONTENT_TYPE'   => 'application/x-www-form-urlencoded',
-            'QUERY_STRING'   => query,
-            'rack.input'     => StringIO.new(body)
+            {
+              'REQUEST_METHOD' => verb.to_s.upcase,
+              'PATH_INFO'      => path,
+              'CONTENT_TYPE'   => 'application/x-www-form-urlencoded',
+              'QUERY_STRING'   => query,
+              'rack.input'     => StringIO.new(body)
+              }.merge(opts[:headers])
```

Comment:
I actually think the `HTTP_` prefix format is being added in the background because that's what I see from the service side. I'll look more into how this is happening